fit/core-reference/src/test/java/org/apache/syncope/fit/core/AnyObjectITCase.java

```diff
@@ -253,4 +256,23 @@ public void issueSYNCOPE1472() {
         assertFalse(printer.getResources().contains(RESOURCE_NAME_DBSCRIPTED), "Should not contain removed resources");
         assertFalse(printer.getAuxClasses().contains("csv"), "Should not contain removed auxiliary classes");
     }
+
+    @Test
+    void issueSYNCOPE1830() {
```

Comment:
Please move this test method to `SearchITCase` and name it accordingly, e.g. `byMembershipAttribute()` or so